packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -172,10 +174,30 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
             occluding some of its content.
          */}
         <ControlBar layout="horizontal" styles={controlBarStyles(theme.semanticColors.bodyBackground)}>
-          {isEnabled(options?.microphoneButton) && <Microphone displayType={options?.displayType} />}
-          {isEnabled(options?.cameraButton) && <Camera displayType={options?.displayType} />}
+          {isEnabled(options?.microphoneButton) && (
+            <Microphone
+              displayType={options?.displayType}
+              /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */ disabled={
+                props.disableForHoldScreen
```

Comment:
Do these show appropriate tooltips when disabled due to being on hold? Do we need to add a new string for these?

Reply:
Discussed offline, just a nice-to-have, can revisit if necessary